### PR TITLE
Add new geophysical attributes accuracy, precision, resolution

### PR DESCRIPTION
**Pull request by daltonkell**
Addresses https://github.com/ioos/compliance-checker/issues/839

### CODE REVIEW

**Comment by daltonkell** [comment]
@ocefpaf It looks like AppVeyor failed to connect to GitHub -- likely a temporary issue.

https://ci.appveyor.com/project/ocefpaf/compliance-checker/builds/34392980

image


I notice that this seems to be your AppVeyor, or is that just how it's set up for the public repo? Don't know much about the AppVeyor configuration.

**Comment by ocefpaf** [comment]
> I notice that this seems to be your AppVeyor, or is that just how it's set up for the public repo? Don't know much about the AppVeyor configuration.

We need to move away from AppVeyor, there is no granularity on the permissions and it is quite slow. Let me check if we should to Travis-CI Windows provider or something else.

File: compliance_checker/ioos.py [hunk 1]
```diff
@@ -740,10 +743,39 @@ def check_geophysical_vars_have_attrs(self, ds):
...
+                if isinstance(val, Number):
+                    r = True
+                else:
+                    r = False
+
+                results.append(
+                    Result(
+                        BaseCheck.MEDIUM,
+                        r,
+                        f"geophysical_variable:{att}",
```
Comment (benjwadams) [inline compliance_checker/ioos.py:767]: Use `str.format()` instead of f-strings for backwards compatibility with older Python 3 versions.

**Review by benjwadams — CHANGES_REQUESTED** [review]

File: compliance_checker/ioos.py (hunk 1 above)
Comment (ocefpaf) [reply compliance_checker/ioos.py:767]: f-strings are new on Python 3.6 and, with most [scientific software adopting NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html), that Python is no longer supported. That bares the question should we support Python <=3.6?

Comment (benjwadams) [reply compliance_checker/ioos.py:767]: Hmmm.  Not that long ago we dropped Python 2.  I wasn't aware of this NEP.  My NumPy version appears to be higher than the Python 3.6 supported version, so it's likely safe.  I don't think we've explicitly pinned a NumPy version, but rather it's going to be dependent on some of the other packages such as NetCDF.  To be on the safe side, I'd say convert to `str.format` for now, with a view towards supporting Python 3.6 as a minimum baseline in the near future.

Comment (benjwadams) [reply compliance_checker/ioos.py]: Going to target 3.6+ from now on, so this is OK.

**Pull request merged**